src/main/java/org/intellij/sdk/codesync/files/CodeSyncYmlFile.java

```diff
@@ -105,12 +105,7 @@ public static void removeFileContents(File ymlFile) {
                     ));
                 }
             }
-            RandomAccessFile randomAccessFile = new RandomAccessFile(ymlFile, "rw");
-            FileChannel fileChannel = randomAccessFile.getChannel();
-            FileLock fileLock = fileChannel.tryLock();
-            if (fileLock != null) {
```

Comment:
What happens if fileLock is null? @syedgulahmed

Reply:
Hi, yeah I reverted the changes and just removed the flush() method, lets see what @saleem-latif is saying.

Reply:
Cool, seems like tests passed on Windows with new change